cluster/charts/rook-minio-objectstore/templates/service.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ template "rook-minio-objectstore.fullname" . }}
+  labels:
+    app: {{ template "rook-minio-objectstore.name" . }}
+    chart: {{ template "rook-minio-objectstore.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+spec:
+  type: {{ .Values.service.type }}
+  ports:
+    - port: {{ .Values.service.port }}
+      targetPort: http
+      protocol: TCP
+      name: http
+  selector:
+    app: {{ template "rook-minio-objectstore.name" . }}
+    release: {{ .Release.Name }}
```

Comment:
Finally with https://github.com/rook/rook/pull/2463 merged, a selector to `objectstore: {{ template "rook-minio-objectstore.fullname" . }}` can be added here to be able to run multiple Minio instances in one namespace.